scripts/requirements.txt

```diff
@@ -6,4 +6,4 @@ jsonschema >= 4.0.0
 
 PyYAML >= 3.12
 requests >= 2.18.4
-towncrier == 21.9.0rc1
+towncrier == 23.6.0
```

Comment:
Note that towncrier's version is pinned here but not in CI, should that be the case?

Reply:
> Note that towncrier's version is pinned here but not in CI, should that be the case?

Yes, it should probably be pinned in CI too. Or rather, to save us inevitably forgetting to update one or the other, CI should `pip install -r requirements.txt` rather than installing a random version of towncrier.